modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -890,7 +890,7 @@ private void _collectObjectFields(
 
 				TreeSet<Method> methodsTreeSet =
 					methodsSortedMap.computeIfAbsent(
-						_getPath(servletData),
+						servletData.getPath(),
```

Comment:
I don't understand why this change is needed, because instead, you are removing the "-graphql" afterwards in each iteration